### PR TITLE
Finalize readme

### LINKED ISSUE #43: Fix readme

### LINKED ISSUE #48: Explain Google Drive remote
Mention more prominently in README and add explanation of how to authorize google drive for dvc (or at least add a link to the dvc page somewhere)

https://dvc.org/doc/user-guide/data-management/remote-storage/google-drive

### PR DESCRIPTION
**Pull request by Timdnb**

### CODE REVIEW

**Review by SagaRut — APPROVED** [review]
Readme looks good

**Comment by Timdnb** [comment]
I'll merge it after Jurriaans branch is merged

**Pull request merged**
